app/controllers/ops_controller/settings/automate_schedules.rb

```diff
@@ -17,10 +17,24 @@ def automate_schedules_set_vars
     }
   end
 
+  def name_included?(klass)
+    klass.safe_constantize.column_names.include?('name')
+  end
+
+  def columns_for_klass(klass)
+    if klass == 'Tenant'
+      [:name, :use_config_for_attributes]
+    elsif name_included?(klass)
+      [:name]
+    else
+      [:description]
+    end
+  end
+
   def fetch_target_ids
     if params[:target_class] && params[:target_class] != 'null'
-      targets = Rbac.filtered(params[:target_class]).select(:id, :name)
-      unless targets.nil?
+      targets = Rbac.filtered(params[:target_class]).select(:id, *columns_for_klass(params[:target_class]))
```

Comment:
What's the `params[:target_class]` contains? Is it coming from the client? Then this thing might be unsafe :confused:

Reply:
I am not sure but also I am not sure if I should change this to something else because this has worked well all the time, and I am just editing it little bit. If you think this needs to be changed, what is your suggestion? How would we make this enough safe?

Reply:
```
[1] pry(#<OpsController>)> params[:target_class]
=> "MiqGroup"
```
for example

Reply:
you are right @skateman and I think we should create a whitelist and check whether `params[:target_class]` is present in the list  but as @hstastna wrote - she is not changing it, so I am suggesting to do it in other PR ...